core-web/apps/dotcms-ui/src/app/view/components/_common/dot-workflows-actions-selector-field/dot-workflows-actions-selector-field.component.ts

```diff
@@ -56,7 +56,7 @@ export class DotWorkflowsActionsSelectorFieldComponent
             tap((actions: SelectItemGroup[]) => {
                 const actionsIds = this.getActionsIds(actions);
 
-                if (actionsIds.length && !actionsIds.includes(this.value)) {
+                if (this.dropdown && actionsIds.length && !actionsIds.includes(this.value)) {
                     this.dropdown.clear(null);
                 }
```

Comment:
The parameter in this function is optional, so it is not necessary to send null instead of sending without parameter like this: `.clear()` is enoght.